src/main/kotlin/no/nav/syfo/service/microfrontend/MikrofrontendDialogmoteService.kt

```diff
@@ -17,7 +17,7 @@ import org.slf4j.LoggerFactory
 class MikrofrontendDialogmoteService(
     val database: DatabaseInterface
 ) {
-    private val log = LoggerFactory.getLogger(MikrofrontendService::class.java)
+    private val log = LoggerFactory.getLogger(MikrofrontendDialogmoteService::class.java.canonicalName)
```

Comment:
Vi har brukt `qualifiedName` mest i appen, så kanskje holde på det i stedet for `java.canonicalName`?
Eks: `private val log = LoggerFactory.getLogger(DokarkivConsumer::class.qualifiedName)`